cmd/main.go

```diff
@@ -36,7 +37,12 @@ func Start() {
 
 	if mockOnlyFlag != nil && *mockOnlyFlag {
 		// Mock-only mode
-		(&mockmode.Main{}).Go()
+		if forcePortFlag != nil && *forcePortFlag > 0 {
```

Comment:
This whole block can just be condensed down to `(&mockmode.Main{}).Go(*forcePortFlag)` the flag defaults to 0 and won't actually be nil (I think...)